experiments/knowlege_distillation/AttentionTransfer.ipynb

```diff
@@ -0,0 +1,3592 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {
+    "id": "CP9FUD9yVTn3"
+   },
+   "outputs": [],
+   "source": [
+    "from trailmet.algorithms.distill.attention_transfer import AttentionTransfer\n",
```

Comment:
For me, somehow trailmet import did not work. I solved by adding 'sys.path.append("../../")' line. Did it work for you?

Reply:
Actually, I kept the notebook in trailmet directory while training and later moved it to the experiments folder.